pkgs/by-name/ne/neovim-unwrapped/package.nix

```diff
@@ -147,7 +147,12 @@ in {
       find "$out" -type f -exec remove-references-to -t ${stdenv.cc} '{}' +
     '';
     # check that the above patching actually works
-    disallowedRequisites = [ stdenv.cc ] ++ lib.optional (lua != codegenLua) codegenLua;
+    outputChecks = let
+      disallowedRequisites = [ stdenv.cc ] ++ lib.optional (lua != codegenLua) codegenLua;
+    in {
+      out = { inherit disallowedRequisites; };
+      debug = { inherit disallowedRequisites; };
```

Comment:
didn't know we had a "debug" output: is that a generic new thing ?

Reply:
No, `separateDebugInfo = true;` causes this (all the debug symbols are written into this output then).